test/files/team.iced

```diff
@@ -1,22 +1,52 @@
 {alloc,team} = require '../../'
-{KeyManager} = require('kbpgp').kb
+{EncKeyManager,KeyManager} = require('kbpgp').kb
 {make_esc} = require 'iced-error'
 {new_sig_arg} = require './util'
 
+test_klass = ({T,arg, klass, keys}, cb) ->
+  esc = make_esc cb, "test_klass"
+  delete arg.kms
+  delete arg.team.per_team_key
```

Comment:
It's just resettting state every time.  It's testing team sigs with and without the `per_team_key` section